dg/config/__init__.py

```diff
@@ -203,5 +203,56 @@ def store_credentials(
         with open(dg_credentials_file_path, "w") as credentials_file:
             json.dump(credentials, credentials_file, indent="\t", sort_keys=True)
 
+    def get_dg_settings_file_path(self) -> pathlib.Path:
+        """Returns the path of the settings file
+
+        Returns
+        -------
+        str
+            path of the settings file
+        """
+
+        return pathlib.Path.home() / ".dg" / "settings.json"
+
+    def show_fyre_options(self) -> bool:
+        """Returns true if Fyre options should be shown
+
+        Returns
+        -------
+        bool
+            true if options should be shown
+            false if not
+        """
+
+        result = False
+
+        if self.get_dg_settings_file_path().exists():
+            settings = json.loads(self.get_dg_settings_file_path().read_text())
+
+            if "show_fyre" in settings and settings["show_fyre"]:
```

Comment:
I would basically use `isHidden` logic in the code and `show_fyre_commands` in the configuration file, doing the negation in this method. To avoid confusion, maybe use a non-boolean field with the values `disabled` and `enabled`.

So the config file would display:
```
fyre_commands : enabled
```
`fyre_commands_hidden`, would return false.